charts/prometheus/templates/prometheus-config-configmap.yaml

```diff
@@ -188,6 +188,20 @@ data:
             regex: 'fluentd_tail_file_.*'
             action: drop
 
+      {{- if .Values.global.nodeExporterEnabled }}
+      - job_name: node-exporter
+        scrape_interval: 30s
+        kubernetes_sd_configs:
+          - role: pod
```

Comment:
This is where `endpoints` would be used if you wanted to use the default helm chart settings